lib/salesforce/client.rb

```diff
@@ -0,0 +1,20 @@
+module Salesforce
+  class Client < Restforce::Data::Client
+
+    def initialize
+      user = SalesforceUser.first
+      if user.nil?
+        Rails.logger.error { "The Salesforce client was requested but no user is available." }
+        raise Salesforce::UserMissing
+      end
+      secrets = SECRET_SETTINGS[:salesforce]
```

Comment:
This will also have to change, probably to `Rails.application.secrets[:salesforce]` or similar.